### PR TITLE
created server and add folders for session and util

### PR DESCRIPTION
**Pull request by srosas**

### CODE REVIEW

**Comment by srosas** [comment]
closes #4

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  userController.getAllUsers((err, users) => {
+    if (err) throw err;
+    res.render('./../client/isLoggedIn', { users: users });
+  });
+});
+
+//Oauth process
+app.get('/oauth', (req, res) => {
+  const code = req.query.code;
+  const clientID = '050e885184cfe2efade9';
```
Comment (kiacolbert) [inline server/server.js]: Delete old values for clientID and ClientSecret

File: server/session/sessionController.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const jwt = require('jsonwebtoken');
```
Comment (kiacolbert) [inline server/session/sessionController.js]: Remove this file from the pull request since you are still working on it. And it's separate issue

**Review by kiacolbert — CHANGES_REQUESTED** [review]

File: server/server.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+const sessionController = require('./session/sessionController')
+const cookieController = require('./util/cookieController');
+
+const app = express();
+
+///////////////////////////////
+//connect to Postgres database here
+////////////////////////////
+
+app.set('view engine', 'ejs');
```
Comment (kiacolbert) [inline server/server.js]: Remove EJS.

**Review by kiacolbert — APPROVED** [review]

**Pull request merged**